deployment/ansible/group_vars/all

```diff
@@ -55,3 +55,5 @@ numba_version: "0.38.1"
 phantomjs_version: "2.1.*"
 
 redis_version: "2:3.0.6-1ubuntu0.*"
+
+default_file_storage: "storages.backends.s3boto3.S3Boto3Storage"
```

Comment:
I am making this storage backend universal since @rajadain's issue called for a development bucket to be created as well.

Reply:
Actually, I think we can use local file storage for development. There is something to be said about having parity between dev and staging / production, but having individual file backends per developer seems better than having a common bucket.

I wouldn't hold back this effort if that's the only thing that remains.

Reply:
Oh I see this has already been implemented, nevermind the above comment.

Reply:
I think that this could be fine left as is, because it seems like the feature is unlikely to get exercised much in development.

With a centralized bucket for development, the challenge will be that any one local development environment will only ever have references to a subset of the objects in the development S3 bucket. In staging and production, the expectation will be that the contents of the S3 bucket and the database will be in-sync.